packages/osd-i18n/src/core/i18n.ts

```diff
@@ -261,5 +261,12 @@ export async function load(translationsUrl: string) {
     throw new Error(`Translations request failed with status code: ${response.status}`);
   }
 
-  init(await response.json());
+  const data = await response.json();
+
+  if (data.warning) {
+    // Store the warning to be displayed after core system setup
+    (window as any).__i18nWarning = data.warning;
```

Comment:
is it common to store this in window?

Reply:
I think it is not uncommon, especially in situations where we need to pass information from sever side to browser that doesn't have a direct communication channel.